docs/0.4/concepts/about_biome.md

```diff
@@ -1,17 +1,422 @@
+Splinter components. Biome stores user and credential information in a local
+PostgreSQL database, manages access tokens, and provides encrypted keys for
+other Splinter actions, such as signing transactions on behalf of a user.
 
-## Modules
+Biome stores information in a local PostgreSQL database on the Splinter node.
+Splinter does not share any user, credential, or key information with other
+Splinter nodes. See [Biome Database Tables](#biome-database-tables) for the
+database structure and table names.
 
-* **User Management**: API to add, delete, update, and retrieve user
-information.
+Biome is part of the Splinter daemon, `splinterd`. The `splinterd` REST API
+exposes this functionality as Biome-specific endpoints. The [splinterd REST API
+Reference](https://www.splinter.dev/docs/0.4/api/) provides documentation and
+examples for the Biome routes and request/response format.
 
-* **Credential Management**: API to register and authenticate a user using a
-username and password. Not recommended for use in production.
+This topic explains Biome's architecture, then illustrates how Biome works with
+examples of using Biome endpoints.
 
-* **Private Key Management**: API to store and retrieve encrypted private keys.
+## Biome Architecture
+
+Biome provides the following functions:
+
+* User account management: Creates and stores users by username and ID.
+  A Splinter user has a username, access credentials, and an automatically
+  generated user ID (UUID). Biome generates the UUID when a user is registered.
+
+* Key management: Encrypts and stores users' private keys.
+
+* Credential management: Registers and authenticates users.  The initial
+  implementation uses usernames and passwords for authentication (not
+  recommended for production use).
+
+* Token management: Stores a refresh token for each user that can be used to
+  issue a new access token when the previous one expires.
+
+* Database migration: Updates the Biome database when a new release adds or
+  changes tables.
+
+### Biome REST API routes
+
+Biome is part of the Splinter daemon, `splinterd`. The `splinterd` REST API
+exposes Biome functionality as Biome-specific endpoints. The following diagram
+shows the relationship of the Biome component and REST API routes with the other
+`splinterd` components and routes.
+
+<img alt="Biome architecture" src="../images/biome_architecture_diagram.svg">
+
+### Key Management in Biome
+
+Biome's user key management provides the capability to store and retrieve
+public/private key pairs and associate them with a user. The Biome
+implementation is a general solution that reduces the overhead of writing a new
+key management system for any application that needs keys. It is designed to be
+flexible enough to support a multitude of use cases in an enterprise
+environment.
+
+The design of Biome includes the following assumptions:
+
+* The front-end application is responsible for encrypting the user's private key
```

Comment:
Is this more of a "client application" than a "front end"?

Reply:
Made this change.